### DISCUSSION TITLE
Trace cable from equipment to final router (through multiple switchs)

### DISCUSSION THREAD

**Discussion by tux-mania** [netbox-community/netbox#10629] [Q&A] [answered]
Hello,

I tried to figure it out on Netbox but couldn't find anything.
I am linking network equipments to each other, but don't know how to view the full cable path from final equipment (for example a patch panel RJ45 to the main router. Even though the links are done between the equipments, when tracing cable on the RJ45 port, it traces only until the next switch. I cannot step further. 
How can I get the full pathway?

Thank you for your help.

**Comment by mtinberg** [comment, marked as answer]
The cable trace feature just shows the path of a physical connection between two devices, which may go through a patch panel, it is not mapping out the spanning tree of a layer2 domain, so it doesn't try to model through switches, so if you had a switch connected to a patch panel which has a cable to a wall jack and a patch cable to a host then you'd have three cables and could trace from the switch interface to the endpoint host, but not from the endpoint host to the router for that VLAN or something.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: tux-mania ***@***.***>
Sent: Wednesday, October 12, 2022 9:20 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Trace cable from equipment to final router (through multiple switchs) (Discussion #10629)


Hello,

I tried to figure it out on Netbox but couldn't find anything.
I am linking network equipments to each other, but don't know how to view the full cable path from final equipment (for example a patch panel RJ45 to the main router. Even though the links are done between the equipments, when tracing cable on the RJ45 port, it traces only until the next switch. I cannot step further.
How can I get the full pathway?

Thank you for your help.

**Reply by tux-mania** [reply]
Thanks for your answer.
